plotly/tools.py

```diff
@@ -244,9 +276,10 @@ def get_config_file(*args):
     """
     if check_file_permissions():
         ensure_local_plotly_files()  # make sure what's there is OK
-        return utils.load_json_dict(CONFIG_FILE, *args)
+        returned_obj = utils.load_json_dict(CONFIG_FILE, *args)
     else:
-        return FILE_CONTENT[CONFIG_FILE]
+        returned_obj = FILE_CONTENT[CONFIG_FILE]
+    return returned_obj
```

Comment:
Is this change necessary? Could you add it to a separate commit with a commit msg that explains why if so?